scene/gui/dialogs.compat.inc

```diff
@@ -0,0 +1,46 @@
+/**************************************************************************/
+/*  scene\gui\dialogs.compat.inc                                          */
```

Comment:
Shouldn't this be without dirs?

Reply:
This should be file only.

Reply:
Could you test this patch?
```diff
diff --git a/misc/scripts/copyright_headers.py b/misc/scripts/copyright_headers.py
index b60eb32289..2144cacc28 100755
--- a/misc/scripts/copyright_headers.py
+++ b/misc/scripts/copyright_headers.py
@@ -41,6 +41,8 @@ fname = sys.argv[1]
 fsingle = fname.strip()
 if fsingle.find("/") != -1:
     fsingle = fsingle[fsingle.rfind("/") + 1 :]
+elif fsingle.find("\\") != -1:
+    fsingle = fsingle[fsingle.rfind("\\") + 1 :]
 rep_fl = "$filename"
 rep_fi = fsingle
 len_fl = len(rep_fl)
```

Reply:
Or simply:
```diff
diff --git a/misc/scripts/copyright_headers.py b/misc/scripts/copyright_headers.py
index b60eb32289..6e2dad6c25 100755
--- a/misc/scripts/copyright_headers.py
+++ b/misc/scripts/copyright_headers.py
@@ -1,6 +1,7 @@
 #!/usr/bin/env python
 # -*- coding: utf-8 -*-
 
+import os
 import sys
 
 header = """\
@@ -38,9 +39,7 @@ header = """\
 fname = sys.argv[1]
 
 # Handle replacing $filename with actual filename and keep alignment
-fsingle = fname.strip()
-if fsingle.find("/") != -1:
-    fsingle = fsingle[fsingle.rfind("/") + 1 :]
+fsingle = os.path.basename(fname.strip())
 rep_fl = "$filename"
 rep_fi = fsingle
 len_fl = len(rep_fl)
```

Reply:
Tested the latter and it works correctly.
btw when using the script for the first time I got out of index error, because I didn't provide a file. It could be handled with proper error message.